crates/toml_edit/src/de/value.rs

```diff
@@ -35,20 +36,18 @@ use crate::de::Error;
 /// # }
 /// ```
 pub struct ValueDeserializer {
-    input: crate::Item,
-    validate_struct_keys: bool,
+    inner: ItemDeserializer,
 }
 
 impl ValueDeserializer {
     pub(crate) fn new(input: crate::Item) -> Self {
```

Comment:
Perhaps it would be better to change `crate::Item` to `crate::Value` here, but I'm not sure if we should introduce more unrelated changes, as it might make the review more difficult. 

If you prefer, I can make this change in a new commit or a separate refactor PR.